docs/advanced/api-v2/node-api.md

```diff
@@ -3,14 +3,12 @@ title: API v2
 ---
 
 FORMAT: 1A
-HOST: https://api.counterparty.io
+HOST: https://api.counterparty.io:4000
 
 # Counterparty Core API
 
 The Counterparty Core API is the recommended (and only supported) way to query the state of a Counterparty node. 
 
-Please see [Apiary](https://counterpartycore.docs.apiary.io/) for interactive documentation.
```

Comment:
Why delete this?

Reply:
oops! not voluntary! will add this sentence in harcoded part!